### ISSUE TITLE
Error occurred in the 'YoastWooCommerce-productTitle' assessment

### ISSUE THREAD

**Issue by mariusghitulescu** [Yoast/wpseo-woocommerce#286]
### What did you expect to happen?

I didn't expect any errors to be shown in the Focus Keyphrase analysis when activating WooCommerce SEO RC1. The issue appears on WooCommerce product pages only. The analysis in product categories and brands works as expected.

### What happened instead?

An error is displayed after activating WooCommerce SEO 9.0 RC1.` The error is shown only for product pages. Product category pages and Brand pages do not have that error in the keyphrase analysis

Error message: `An error occurred in the 'YoastWooCommerce-productTitle' assessment`

![2018-10-18 11_49_55-window](https://user-images.githubusercontent.com/4171016/47142644-0c84ef00-d2cc-11e8-98de-bd354e98c637.png)

### How can we reproduce this behavior?

1. While using Yoast SEO 9.0 RC3, create a product page and set a focus keyphrase.
2. Check the analysis for the focus keyphrase, it should work properly.
3. Activate WooCommerce SEO 9.0 RC1, the analysis should show the error from the screenshot.

Can you provide a link to a page which shows this issue?

It's on a local environment, but I can try to replicate it on a live website if needed.

### Technical info
* WordPress version: 4.9.8
* Yoast SEO version: free and premium 9.0 RC3

**Comment by mariusghitulescu** [comment]
I started with the a similar environment, from Yoast SEO (free and premium) 8.4 and WooCommerce SEO 8.4, then gradually updated them one by one to see exactly when the error shows up. 

I have not been able to replicate it. I guess it was something specific to my first testing environment.

**Comment by andizer** [comment]
I think this is duplicate of: https://github.com/Yoast/YoastSEO.js/issues/1835 (which seems to be posted in wrong repo?)

**Comment by mariusghitulescu** [comment]
@andizer you are correct, I didn't check that repo. Also the error does indeed show up when using the Text editor only. Switching to Visual fixes it.

**Comment by IreneStr** [comment]
Good to know: despite its name, this assessment actually assesses the product description.

**Comment by IreneStr** [comment]
Info from https://github.com/Yoast/YoastSEO.js/issues/1835

- Important: this error only occurs in the text editor (not the visual editor). Switch to the Text editor and save the page
- These console errors are shown:
![image](https://user-images.githubusercontent.com/9658774/46471402-02e28e00-c7da-11e8-9fe5-4d1dd6703695.png)

**Comment by Pcosta88** [comment]
Please inform the customer of conversation # 435561 when this conversation has been closed.

**Comment by monbauza** [comment]
Please inform the customer of conversation # 431009 when this conversation has been closed.

**Comment by monbauza** [comment]
Please inform the customer of conversation # 437024 when this conversation has been closed.

**Comment by mayada-ibrahim** [comment]
Please inform the customer of conversation # 429588 when this conversation has been closed.
